core/state_processor.go

```diff
@@ -234,6 +234,6 @@ func ApplyTransaction(
 	}
 	// Create a new context to be used in the EVM environment
 	blockContext := NewEVMBlockContext(header, bc, author, publishEvents...)
-	vmenv := vm.NewEVM(blockContext, vm.TxContext{}, statedb, config, cfg)
+	vmenv := vm.NewEVM(blockContext, vm.TxContext{BlobHashes: tx.BlobHashes()}, statedb, config, cfg)
```

Comment:
This seems redundant, in `applyTransaction` the new tx context is created this is overwritten by the newly created one.